Portfolio/index.html

```diff
@@ -1,16 +1,116 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>Portfolio</title>
-    <meta charset="UTF-8" />
-    <link rel="stylesheet" href="index.css" />
-  </head>
-
-  <body style="margin: 0; padding: 0">
-    <div top="0">
-      <a href="../index.html">Go Back</a>
-    </div>
-    <!-- Write your code below for PORTFOLIO and DO NOT TOUCH the code above-->
-    <!-- to import image from public folder, do it like this= <img src="../public/css.icon.png" alt="CSS image"> -->
-  </body>
-</html>
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Portfolio</title>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="index.css" />
+  </head>
+
+  <body style="margin: 0; padding: 0">
+    <div top="0">
+      <a href="../index.html">Go Back</a>
+    </div>
+    <!-- Write your code below for PORTFOLIO and DO NOT TOUCH the code above-->
+    <!-- to import image from public folder, do it like this= <img src="../public/css.icon.png" alt="CSS image"> -->
+  </body>
+</html>
+<!DOCTYPE html>
+<html>
+    <head>
+        <title> <ul>PORTFOLIO</tittle>
+        <li>ABOUT</li>
+        <li>SKILL</li>
+        <li>PROJECTS</li>
+        <li>REVIEWS</li>
+        <br><br>
+        <body>
+        <main>  

```

Comment:
The class attribute values for the 'i' elements are not valid. The prefix 'fa-brands' should be separated by a space from the icon name 'fa-html5', 'fa-css3-alt', 'fa-js', 'fa-react'.